garage/tf/algos/batch_polopt.py

```diff
@@ -1,13 +1,37 @@
-from dowel import logger
+from collections import deque
```

Comment:
please only import stdlib dependencies using their full package, e.g.

```python
import collections

q = collections.deque(5)
```

this make it easier to understand what types are from dependencies when people are reading the code.